hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCLIUtils.scala

```diff
@@ -70,4 +70,16 @@ object HoodieCLIUtils extends ProvidesHoodieConfig{
         throw new SparkException(s"Unsupported identifier $table")
     }
   }
+
+  def getHoodieDatabaseAndTable(table: String): (String, Option[String]) = {
+    val seq: Seq[String] = table.split('.')
```

Comment:
`getHoodieDatabaseAndTable` -> `getTableIdentifier`, the returned val should be a string array

Reply:
done